### PR TITLE
Clear of old map for incident maps.

### PR DESCRIPTION
**Pull request by tomeksabala**
Forgot about incident maps available for Madagascar.

### CODE REVIEW

**Comment by tomeksabala** [comment]
There are some Snyx complains. I'm going to investigate it and ask for review afterwards. @jonathansberry

**Comment by tomeksabala** [comment]
I don't think that there are any solutions to the `requirements.txt` snyk vulnerabilities at the moment. This PR is not changing any dependencies. I suggest we merge it as it is.
I've applied snyk autofixes for JS libraries.
@jonathansberry Please approve if you agree with me.

File: package.json [hunk 1]
```diff
@@ -50,5 +52,6 @@
     "karma-phantomjs-launcher": "1.0.0",
     "karma-spec-reporter": "0.0.22",
     "phantomjs-prebuilt": "2.1.14"
-  }
+  },
+  "snyk": true
```
Comment (jonathansberry) [inline package.json]: This line can cause problem in deployment.

File: package.json [hunk 2]
```diff
@@ -8,37 +8,39 @@
...
-    "snyk": "^1.69.5",
+    "snyk": "^1.72.0",
     "tree-model": "1.0.4",
     "xtend": "4.0.0",
     "yargs": "3.15.0"
   },
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start karma.conf.js --single-run"
+    "test": "./node_modules/karma/bin/karma start karma.conf.js --single-run",
+    "snyk-protect": "snyk protect",
```
Comment (jonathansberry) [inline package.json]: This line can cause problem in deployment.

File: package.json [hunk 3]
```diff
@@ -8,37 +8,39 @@
...
+    "prepublish": "npm run snyk-protect"
```
Comment (jonathansberry) [inline package.json]: This line can cause problem in deployment.

**Comment by jonathansberry** [comment]
Just to clarify.  Snky makes some auto updates to the package dependancies list - which if they work, are good and should be included.  The changes in https://github.com/meerkat-code/meerkat_frontend/commit/5802e03a8ab8ccb0f30b0a9097b81cb9250323b5 do not include the dependancy list.  Jsut some further scripts that snyk adds to the javascript testing.

**Pull request merged**